### DISCUSSION TITLE
How to disable markdown?

### DISCUSSION THREAD

**Discussion by Aymkdn** [ueberdosis/tiptap#850] [General Questions] [answered]
Hi,

I don't want that `_` transforms my words to _italics_. I searched around but I cannot figure out the solution: **how to disable markdown?**

I'm using an old version (v1.13.0).

Thanks!

**Comment by hanspagel** [comment]
You can copy all extensions to your project folder, remove the inputRules (that’s what they are for) and register them with tiptap. Hope that helps!

**Comment by Aymkdn** [comment, marked as answer]
Actually, in the version I'm using, I used `disablePasteRules` and `disableInputRules`, and it looks like it worked:
```js
this.editor = new Editor({
      content: "",
      disablePasteRules:true, // disable Markdown when pasting
      disableInputRules:true, // disable Markdown when typing
      extensions:[
        new Bold(),
        new Italic(),
        new OrderedList(),
        new BulletList(),
        new ListItem(),
        new Link(),
        new History()
      ],
      onUpdate:obj => {
        // do something with obj.getHTML()
      }
    })
```

**Reply by hanspagel** [reply]
Oh yes, that’s way easier. 😅 Forgot about those options.

**Reply by rafaelrollup** [reply]
Just an update, these params have changed, they're now:

[enablePasteRules](https://tiptap.dev/docs/editor/api/editor#enablepasterules)
[enableInputRules](https://tiptap.dev/docs/editor/api/editor#enableinputrules)
